chainweb.cabal

```diff
@@ -388,6 +392,7 @@ test-suite chainweb-tests
         , extra >= 1.6
         , fgl >= 5.6
         , hashable >= 1.2
+        , hashtables >= 1.2
```

Comment:
Just use `IORef` + `HashMap`? This is for the testsuite and the additional dependency doesn't seem worth it to gain speedup there.